lib/utilities/aclUtils.js

```diff
@@ -233,4 +234,72 @@ aclUtils.sortHeaderGrants =
         return addACLParams;
     };
 
+/**
+ * convertToXml - Converts the `grantInfo` object (defined in `objectGetACL()`)
+ * to an XML DOM string
+ * @param {object} grantInfo - The `grantInfo` object defined in
+ * `objectGetACL()`
+ * @return {string} xml.join('') - The XML DOM string
+ */
+aclUtils.convertToXml = grantInfo => {
+    const { grants, ownerInfo } = grantInfo;
+    const xml = [];
+
+    // Escape any special XML characters
+    for (const key in grants) {
+        if (grants.hasOwnProperty(key)) {
+            grants[key] = escapeForXML(grants[key]);
+        }
+    }
+    for (const key in ownerInfo) {
+        if (ownerInfo.hasOwnProperty(key)) {
+            ownerInfo[key] = escapeForXML(ownerInfo[key]);
+        }
+    }
+
+    // Begin pushing XML element strings to the `xml` to create the XML DOM
+    xml.push('<?xml version="1.0" encoding="UTF-8" standalone="yes"?>',
+             '<AccessControlPolicy>',
+             '<Owner>',
+             `<ID>${ownerInfo.ID}</ID>`,
+             `<DisplayName>${ownerInfo.displayName}</DisplayName>`,
+             '</Owner>',
+             '<AccessControlList>'
+    );
+
+    // For each object in `grants`, push the relevant XML elements to `xml`
```

Comment:
This comment is redundant. Ideally, comments should explain `why` of the code than the `what`.